builtin/string.go

```diff
@@ -350,7 +352,7 @@ func string_NumberQ(this Value) Value {
 }
 
 // benchmark shows Suneido regex is faster than Go regexp for this
-var numberPat = regex.Compile(`\A[+-]?(\d+(\.\d*)?|\.\d+)([eE][+-]?\d\d?)?\Z`)
```

Comment:
I thought we were going to switch to the lexer approach?

Reply:
Sorry, discrepancies between my home computer and work computer.